forte/mrdsrg-spin-integrated/master_mrdsrg.cc

```diff
@@ -48,7 +47,9 @@ void MASTER_DSRG::startup() {
     print_h2("Multireference Driven Similarity Renormalization Group");
 
     // build fock using ForteIntegrals and clean up JK
-    build_fock_from_ints(ints_);
+    if (foptions_->get_bool("EMBEDDING_JKFOCK")) {
```

Comment:
My criticisms of `EMBEDDING_JKFOCK` in the spin-adapted code carry over to here as well.